packages/interop-tests/jest/jest.config.js

```diff
@@ -4,14 +4,17 @@ const root = resolve(__dirname, '../');
 module.exports = {
   rootDir: root,
   testMatch: ['<rootDir>/**/__test__/**/?(*.)test.ts'],
-  testEnvironment: 'jsdom',
+  testEnvironment: 'node',
```

Comment:
Good catch!

Reply:
See PR description for the reason we had to do this.

Reply:
yep, I appreciated the clarification in the commit message as well.